lib/cisco_node_utils/cmd_ref/vlan.yaml

```diff
@@ -16,8 +16,7 @@ mapped_vni:
     kind: int
     config_get: 'show running vlan'
     config_get_token: ['/^vlan <vlan>$/', '/^vn-segment (\d+)$/']
-    # In the cli 'vn-segment' config is not applied untill the 'end'
-    # statement is issued.
+    # The 'vn-segment' config is not applied until the 'end' statement is processed
```

Comment:
Since you're touching this - is it true? Does this feature stop working if you remove the literal `; end` from the YAML?